spec/stringProcessing/syllables/countSpec.js

```diff
@@ -163,18 +768,567 @@ describe( "a syllable counter for Dutch textstrings", function () {
+		expect( countSyllableFunction( "overdressed", "de_DE" ) ).toBe( 3 );
+		expect( countSyllableFunction( "mixed", "de_DE" ) ).toBe( 1 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable aau", function () {
+		expect( countSyllableFunction( "extraausgabe", "de_DE" ) ).toBe( 5 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable a[äöüo]", function () {
+		expect( countSyllableFunction( "klimaänderung", "de_DE" ) ).toBe( 5 );
+		expect( countSyllableFunction( "aöde", "de_DE" ) ).toBe( 3 );
+		expect( countSyllableFunction( "gammaübergang", "de_DE" ) ).toBe( 5 );
+		expect( countSyllableFunction( "aorta", "de_DE" ) ).toBe( 3 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable äue", function () {
+		expect( countSyllableFunction( "abgesäuert", "de_DE" ) ).toBe( 4 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable äeu", function () {
+		expect( countSyllableFunction( "mäeutisch", "de_DE" ) ).toBe( 3 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable aei", function () {
+		expect( countSyllableFunction( "kameraeinstellung", "de_DE" ) ).toBe( 6 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable aue", function () {
+		expect( countSyllableFunction( "mauer", "de_DE" ) ).toBe( 2 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable aeu", function () {
+		expect( countSyllableFunction( "Gammaeule", "de_DE" ) ).toBe( 4 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable ael", function () {
+		expect( countSyllableFunction( "ismael", "de_DE" ) ).toBe( 3 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable ai[aeo]", function () {
+		expect( countSyllableFunction( "kaianlage", "de_DE" ) ).toBe( 4 );
+		expect( countSyllableFunction( "dubaier", "de_DE" ) ).toBe( 3 );
+		expect( countSyllableFunction( "gaio", "de_DE" ) ).toBe( 2 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable saik", function () {
+		expect( countSyllableFunction( "mosaik", "de_DE" ) ).toBe( 3 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable aismus", function () {
+		expect( countSyllableFunction( "archaismus", "de_DE" ) ).toBe( 4 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable ä[aeoi]", function () {
+		expect( countSyllableFunction( "mäandrisch", "de_DE" ) ).toBe( 3 );
+		expect( countSyllableFunction( "achäer", "de_DE" ) ).toBe( 3 );
+		expect( countSyllableFunction( "äolisch", "de_DE" ) ).toBe( 3 );
+		expect( countSyllableFunction( "achäisch", "de_DE" ) ).toBe( 3 );
+	} );
+
+	it( "returns the number of syllables of words containing the add syllable auä", function () {
+		expect( countSyllableFunction( "blauäugig", "de_DE" ) ).toBe( 3 );
```

Comment:
This isn't covered by the fragment `uäu` but by `auä`. So either the `uäu` is redundant, or it should be tested with another word.